HelpSource/Classes/HistoryGui.schelp

```diff
@@ -19,6 +19,99 @@ DEFINITIONLIST::
 ## listview || shows either all code::lineShorts::, or the filtered code::lineShorts::; selecting them in listview makes them come appear in post-doc window.
 ::
 
+first example:
+code::
+(
+h = History.new.lines_([
+	[0, \me, "1+2"],
+	[1, \me, "3+5"], [1.234, \you, "q = q ? ();"],
+	[3, \her, "\"Herstory\".speak"]
+]);
+g = h.makeWin(0@20);
+g.findDoc;
+g.postDoc(2);
+)
+::
+
+ClassMethods::
+
+method::docHeight
+get and set height of textdoc/window opened by HistoryGui
+method::docTitle
+get and set title of textdoc/window opened by HistoryGui
+
+method::new
+create a new HistoryGui
+argument:: object
+the history
+argument:: numItems
+the number of lines in the textview above
+argument:: parent
+a parent window or view in which to place new HistoryGui
+argument:: bounds
+bounds for new HistoryGui, constrained by its minSize.
+argument:: makeSkip
+flag whether to create a skipjack for it.
+argument:: options
+options to set for the gui.
+
+
+InstanceMethods::
+
+private:: accepts, checkUpdate, getState, makeViews, setDefaults, postInlined
+private:: updateBinVal, updateFunc, updateLines, updateVal, alignDoc
+subsection:: views
+method::textV
+top: the textview for the selected line
```

Comment:
please also capitalize `TextView` and `ListView` throughout

Reply:
done.